### PR TITLE
fcosKola: add `build` and `platformArgs` parameters

### PR DESCRIPTION
**Pull request by jlebon**
```
commit e757aa46f0cc60e2ab572cb6f3da5f0bcda2cdfb
Date:   Thu Oct 22 15:24:15 2020 -0400

    fcosKola: add `build` parameter

    Make this a separate parameter instead of having to pass it through
    `extraArgs` because it needs to be used in both `run` and
    `run-upgrade`.

commit 937721dd79593d79e22dda3982e400138a1ecbc4
Date:   Thu Oct 22 15:25:06 2020 -0400

    fcosKola: add `platformArgs` parameter

    Similarly to the previous commit, add a separate `platformArgs`
    parameter since we want to pass this to `kola` in both the `run` and
    `run-upgrade` branches.
```

### CODE REVIEW

File: vars/fcosKola.groovy [hunk 1]
```diff
@@ -32,7 +38,7 @@ def call(params = [:]) {
             def parallel = params.get('parallel', 8);
             def extraArgs = params.get('extraArgs', "");
             try {
-                shwrap("cd ${cosaDir} && cosa kola run --parallel ${parallel} ${args} ${extraArgs}")
+                shwrap("cd ${cosaDir} && cosa kola run --build ${build} ${platformArgs} --parallel ${parallel} ${args} ${extraArgs}")
```
Comment (dustymabe) [inline vars/fcosKola.groovy:41]:
```suggestion
                shwrap("cd ${cosaDir} && cosa kola run --build ${buildID} ${platformArgs} --parallel ${parallel} ${args} ${extraArgs}")
```

File: vars/fcosKola.groovy [hunk 2]
```diff
@@ -45,7 +51,7 @@ def call(params = [:]) {
         kolaRuns['run_upgrades'] = {
             stage("run-upgrade") {
                 try {
-                    shwrap("cd ${cosaDir} && cosa kola --upgrades")
+                    shwrap("cd ${cosaDir} && cosa kola --upgrades --build ${build} ${platformArgs}")
```
Comment (dustymabe) [inline vars/fcosKola.groovy:54]:
```suggestion
                    shwrap("cd ${cosaDir} && cosa kola --upgrades --build ${buildID} ${platformArgs}")
```

File: vars/fcosKola.groovy (hunk 1 above)
Comment (dustymabe) [reply vars/fcosKola.groovy:41]: or change `def buildID` to `def build` above

Comment (jlebon) [reply vars/fcosKola.groovy]: Nice catch, thanks!

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**